app/controllers/users_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
```

Comment:
- [ ] The same way as the #create the #update action has two possible outcomes, please add an else in case it can't update.
- redirect_to edit_user_path if user was saved
- render :edit if user was not saved

Reply:
An if/else statement has now been implemented in UsersController#update, similarly to #create.